tools/config_json_validator.py

```diff
@@ -0,0 +1,26 @@
+import subprocess
+import os
+import json
+
+
+result = subprocess.run(['find', '../', '-name', 'test-config.json'], capture_output=True, text=True)
```

Comment:
Before this line, do this:

```
os.chdir(Path(__file__).parent)
```

This is the ensure that the working directory for the script is the script's actual parent path.

Otherwise your `find ..` command won't work as expected for people invoking the script from the root directory instead of `cd`ing into the tools directory first (a lot of people don't cd all the way into a directory to invoke a script and they shouldn't have to).

Reply:
Also consider using glob instead of find:
`glob.glob("../**/**/test-config.json")`